### ISSUE TITLE
Create default networkpolicies pr app

### ISSUE THREAD

**Issue by jhrv** [nais/naiserator#108]
Today we create a default set of networkpolicies for the entire namespace when we create the namespace. This should be done for each application.

For GCP, we need to reach pilot, mixer and kube-dns.
This should be configurable in so it's not tightly bound to our own use-case.

```
apiVersion: extensions/v1beta1
kind: NetworkPolicy
metadata:
  name: myapp-default-egress-allow
  namespace: thenamespace
spec:
  egress:
  - to:
    - namespaceSelector:
        matchLabels:
          name: istio-system
      podSelector:
        matchLabels:
          istio: pilot
    - namespaceSelector:
        matchLabels:
          name: istio-system
      podSelector:
        matchLabels:
          istio: mixer
    - namespaceSelector:
        matchLabels:
          name: kube-system
      podSelector:
        matchLabels:
          k8s-app: kube-dns
  - to:
    - ipBlock:
        cidr: 0.0.0.0/0
        except:
        - 10.0.0.0/8
        - 172.16.0.0/12
        - 192.168.0.0/16
  podSelector: 
    matchLabels:
      app: myapp
  policyTypes:
  - Egress
```

**Comment by linemos** [comment]
Trenger vi ikke ingress fra mixer i istio-system også?

**Comment by jhrv** [comment]
hm, kaller mixer appen?

**Comment by linemos** [comment]
Hm.. Nei nå er det nok jeg som surrer.

**Comment by linemos** [comment]
Solved by #122
